config/application.rb

```diff
@@ -31,5 +31,21 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Lets fix the generators. What is and what is not going to be generated by 'rails g'
+    config.generators do |g|
+      g.stylesheets false
+      g.javascripts false
+      g.test_framework :rspec,
+                       fixtures: true,
+                       model_specs: true,
+                       view_specs: false,
+                       helper_specs: true,
+                       routing_specs: false,
```

Comment:
I haven't had much exposure to testing so far to form an opinion as to what we should and should not test in the application.